jerry-libc/include/stdlib.h

```diff
@@ -28,9 +28,15 @@ extern "C"
  *          rand
  */
 #define RAND_MAX (0x7fffffffu)
+#ifdef _MSC_VER
+#define NO_RETURN_ATTRIBUTE __declspec(noreturn)
+#else
+#define NO_RETURN_ATTRIBUTE __attribute__ ((noreturn))
+#endif
 
-void __attribute__ ((noreturn)) exit (int);
-void __attribute__ ((noreturn)) abort (void);
+
+void NO_RETURN_ATTRIBUTE exit (int);
+void NO_RETURN_ATTRIBUTE abort (void);
```

Comment:
If jerry-libc gets disabled for MSVC by default, this change will become unnecessary.